README.md

```diff
@@ -230,27 +230,31 @@ following the ASF Policy.
 
 - **DAGs**: Overview of all DAGs in your environment.
 
-  ![DAGs](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/dags.png)
+  ![DAGs](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/ui-dark/dags.png)
+
+- **Assets**: Overview of Assets with dependencies.
+
+  ![Asset Dependencies](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/ui-dark/assets_graph.png)
 
 - **Grid**: Grid representation of a DAG that spans across time.
 
-  ![Grid](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/grid.png)
+  ![Grid](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/ui-dark/grid.png)
 
 - **Graph**: Visualization of a DAG's dependencies and their current status for a specific run.
 
-  ![Graph](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/graph.png)
+  ![Graph](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/ui-dark/graph.png)
 
 - **Home**: Summary statistics of your Airflow environment.
 
-  ![Home](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/home.png)
+  ![Home](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/ui-dark/home.png)
 
 - **Backfill**: Backfilling a DAG for a specific date range.
 
-  ![Backfill](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/backfill.png)
+  ![Backfill](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/ui-dark/backfill.png)
 
 - **Code**: Quick way to view source code of a DAG.
 
-  ![Code](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/code.png)
+  ![Code](https://raw.githubusercontent.com/apache/airflow/main/airflow-core/docs/img/ui-dark/code.png)
```

Comment:
I assume this are just moving files rather than new files?

Reply:
I took some new screenshots as the once taken before had no "light" edition besides the "dark" as requested to be standard. Hope this is okay as well.

If you insist on the old ones, I could keep the dark ones but then they are different tot the light ones... or the examples used to capture the screens need to be shared, they are not within the normal examples :-D

Reply:
I am good with those as long as supporting text (wherever applicable) is also correct :)